### DISCUSSION TITLE
Compatibility with OpenMCT and COSMOS

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1374] [Q&A]
Hello,

Can we run this FPrime software with OpenMCT and Cosmos ?

**Comment by rawinza555** [comment]
Yes. There's a repo on making openMCT works with F' [here](https://github.com/fprime-community/fprime-openmct-integration) but it is quite old so it requires modification. Three main things you need to do is to supply openMCT & COSMOS with the telemetry you have (Could be extracted from appdictionary.xml), add the function to deframe the incoming packets and connect to the F' itself.

**Reply by ThibFrgsGmz** [reply]
There is an analogous repo for COSMOS: https://github.com/fprime-community/fprime-cosmos-integration.
